platform/security/idp/security-idp-client/pom.xml

```diff
@@ -206,17 +212,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.59</minimum>
+                                            <minimum>0.40</minimum>
```

Comment:
❓ Ouch! Anything we can do to bring those numbers back up?

Reply:
Must have been a JaCoCo run. I just check it again and coverage has gone up, not down. Look for new numbers in the next commit.